src/main/java/io/github/moulberry/notenoughupdates/core/config/gui/GuiOptionEditorDropdown.java

```diff
@@ -87,6 +88,7 @@ public void renderOverlay(int x, int y, int width) {
 
             TextRenderUtils.drawStringScaledMaxWidth(selectedString, fr, left + 3, top + 3, false,
                     dropdownWidth - 16, 0xffa0a0a0);
+            GlStateManager.translate(0, 0, -100);
```

Comment:
In general i prefer using pushState and popState but if its just translation then the performance impact is minimal and theres prolly no one injecting into our code